src/Medidata.ZipkinTracer.Core/ZipkinConfig.cs

```diff
@@ -49,5 +49,15 @@ public List<string> GetNotToBeDisplayedDomainList()
 
             return zipkinNotToBeDisplayedDomainList;
         }
+
+        public Uri GetZipkinProxyServer()
+        {
+            var proxy = System.Web.Configuration.WebConfigurationManager.GetSection("system.net/defaultProxy") as System.Net.Configuration.DefaultProxySection;
+            if (proxy != null && proxy.Enabled)
+            {
+                return proxy.Proxy.ProxyAddress;
+            }
+            return null;
+        }
```

Comment:
It might be better to use `WebProxy.DefaultWebProxy`

```
public Uri ZipkinProxyServer
{
    get
    {
        if (WebRequest.DefaultWebProxy != null)
        {
            return WebRequest.DefaultWebProxy.GetProxy(new Uri(string.Join(ZipkinServerName, ":", ZipkinServerPort));
        }

        return null;
    }
}
```